com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -83,6 +83,14 @@ gravity_y.type = number
 gravity_y.help = world gravity along y-axis, -10 by default (natural gravity)
 gravity_y.default = -10
 
+velocity_iteration.type = number
+velocity_iteration.help = modify amount of iteration for velocity in Box2D physics
+velocity_iteration.default = 16
+
+position_iteration.type = number
+position_iteration.help = modify amount of iteration for position in Box2D physics
+position_iteration.default = 16
```

Comment:
These names can be clearer imho.
They sound like a boolean.
Perhaps "num_position_iterations" is clearer?